src/api/tests/protocol.rs

```diff
@@ -0,0 +1,23 @@
+use crate::api::protocol::Protocol;
+use crate::api::tests::get_test_versions;
+use crate::models::protocol::PacketDataType;
+use std::sync::Arc;
+use crate::DataResult;
+
+#[test]
+pub fn simple_test() {
```

Comment:
Can you make this one more descriptive and make it fail on error?

Reply:
I will be adding more tests as I add api methods

Reply:
Okay perfect. I assume this is just a placeholder for some quick testing